### ISSUE TITLE
Make python/test/test_monkey pass under Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5816]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)| @vperic reported|
|-|-|
|Trac ID|trac#5816|
|Type|task|
|Created|2012-07-28 09:19:21Z|
|Branch|https://github.com/twisted/twisted/tree/test_monkey-5816|

As the title says, make test_monkey pass under Python 3. To actually run these under Python 3, the import of unittest from twisted.trial needs to be changed to import from the main library (as twisted.trial brings in a *lot* of other dependencies, see [#5815](https://github.com/twisted/twisted/issues/5815) for some more info).

The change itself is really simple, just update one raise statement to use the newer syntax.

Attachments:

* [test_monkey.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/38d/38dfc442cabe3dc21e89cd0ddae73e780377a01a/e41a83331b5fe7aceb85c6e5f4792798c6783639.patch) (623 bytes) - added by vperic on 2012-07-28 09:19:42Z - First version

<details><summary>Searchable metadata</summary>

```
trac-id__5816 5816
type__task task
reporter__vperic vperic
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_test_monkey_5816 branches/test_monkey-5816
branch_author__vperic vperic
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1343467161000000 1343467161000000
changetime__1343816216000000 1343816216000000
version__None None
owner__vperic vperic

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_1"></a>|@itamarst set owner to @vperic|
|-|-|

Looks good, merge.

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_2"></a>|@vperic commented|
|-|-|

(In [35018]) Create branch test_monkey-5816

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_3"></a>|@vperic commented|
|-|-|

[Build results, just for test_monkey.py](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/test_monkey-5816)

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_4"></a>|@vperic set status to `closed`|
|-|-|

(In [35020]) Merge test_monkey-5816: Make t.test.test_monkey pass under Python 3

Author: vperic
Reviewer: itamar
Fixes: [#5816](https://github.com/twisted/twisted/issues/5816)

Update one raise statement to the more modern syntax to make the tests
pass under both Python 2 and 3.
